### ISSUE TITLE
mDNS support

### ISSUE THREAD

**Issue by kpostekk** [WeeJeWel/wg-easy#175]
Is it possible to implement mDNS forwarding OOTB?

**Comment by WeeJeWel** [comment]
I would love this. Does anyone know how?

**Comment by kpostekk** [comment]
_tl;dr use some docker image (probably by docker-compose) to reflect broadcasts and enable multicast and allow some ip range if not using `0.0.0.0` (`224.0.0.251/32` `ff02::fb/128`)._

I have found [this](https://hub.docker.com/r/flungo/avahi) docker image with avahi server. README says it can reflect mDNS broadcasts between networks.
Also I have found [this](https://www.reddit.com/r/WireGuard/comments/g80bxf/cant_get_zeroconfavahimdns_reflection_to_work/) reddit post about enabling multicast in WG.

So I think changing wg init to something like this

```
module.exports.WG_POST_UP = process.env.WG_POST_UP || `
iptables -t nat -A POSTROUTING -s ${module.exports.WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o eth0 -j MASQUERADE;
iptables -A INPUT -p udp -m udp --dport 51820 -j ACCEPT;
iptables -A FORWARD -i wg0 -j ACCEPT;
iptables -A FORWARD -o wg0 -j ACCEPT;
ip link set multicast on dev wg0;
`.split('\n').join(' ');
```

For now i don't have any particular idea for changes in docker-compose

**Comment by kpostekk** [comment]
I will try prototyping something this weekend

**Comment by WeeJeWel** [comment]
AFAIK WireGuard doesn't support this yet.

**Comment by cerahmed** [comment]
Any update on this?

Using the default WG installation I was able to follow instructions in [this ](https://www.reddit.com/r/WireGuard/comments/g80bxf/cant_get_zeroconfavahimdns_reflection_to_work/)link alongside using [mdns-repeater](https://github.com/geekman/mdns-repeater) to get mdns to work over WG network.

However, having a docker container with a vlan (docker0) pointing to wg0, I'm not sure how I can get it to work with wg-easy.
Any idea would be appreciated.

**Comment by fra87** [comment]
Was anything ever done to support mDNS/avahi/zeroconf on wg-easy?
